File: Mr.SikiruTask/class-exercises/transaction_balance.py
print(" WELCOME TO PAFEKZY MOBILE BANK APP!")
 


balance = 0

while True:
    transaction = input("Type in your choice of transaction (deposit or withdraw), or 'quit' to stop: ")
    if transaction.lower() =='quit':
        break

    amount = int(input("Enter amount: "))
    if transaction.lower() == 'deposit':
        balance += amount
        if balance <0 :
            print(" Insufficient Balance. Try Again Later !")
            break

    elif transaction.lower() == 'withdraw':
        balance -= amount
        if balance <0 :
            print(" Insufficient Balance. Try Again Later !")
            break

    print(f"Balance: {balance}")

print(f"Final Balance: {balance}")





















#1. Initialize the balance to 0.
#2. Enter a loop that continues until the user enters 'quit'.
#3. Ask the user to enter a transaction eigher deposit / withdraw.
#4. Ask the user to enter the amount.
#5. If the transaction is a deposit, add the amount to the balance.
#6. If the transaction is a withdrawal, subtract the amount from the balance.
#7. Print the updated balance.
#8. Repeat steps 3-7 until the user enters 'quit'.
#9. Print the final balance.

 

File: Mr.SikiruTask/python_to_javawork/TransactionBalance.java


import java.util.Scanner;

public class TransactionBalance {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double balance = 0;

        while (true) {
            System.out.print("Enter transaction (deposit/withdraw), or 'quit' to stop: ");
            String transaction = scanner.next();
            if (transaction.equalsIgnoreCase("quit")) {
                break;
            }
            System.out.print("Enter amount: ");
            double amount = scanner.nextDouble();
            if (transaction.equalsIgnoreCase("deposit")) {
                balance += amount;
            } else if (transaction.equalsIgnoreCase("withdraw")) {
                balance -= amount;
            }
            System.out.println("Balance: " + balance);
        }
        System.out.println("Final Balance: " + balance);
    }
}

 
